docs/endpoints/Legacy/Two/posts.md

```diff
@@ -120,7 +120,7 @@ POST /api/v2/posts
   - **source**: (string).
     Where the post was submitted from.
   - **remote_addr**: (string).
-    IP address of where the post is submitted from. 
+    IP address of where the post is submitted from. Will be `000.000.00.00` for all posts in compliance with GDPR. 
```

Comment:
I think it's safe to remove "IP address of where the post is submitted from." since that will no longer be true!